src/status_im2/contexts/scan/view.cljs

```diff
@@ -0,0 +1,39 @@
+(ns status-im2.contexts.scan.view
```

Comment:
since this pr: https://github.com/status-im/status-mobile/pull/16395/files#diff-3cd2c233d3652778b11f61da438495b0b08971297ad7448f126c61d6e3927036

has been merged. You can now put this file in the namespace
`status-im2.contexts.shell.scan.view`

Reply:
Thanks, I have updated the namespace for shell scan